### PR TITLE
Add GenereateSeq function

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: generate-sequence
🏅 Granted 5 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/generate-sequence/generateSequence.hpp [hunk 1]
```diff
@@ -3,5 +3,18 @@
...
 std::vector<int> generateSequence(int count, int step) {
     // TODO: Implement me :)
-    return {};
+    if(count <= 0)
+    {
+        return {};
+    }
+    std::vector<int> returnedVec{};
+    returnedVec.reserve(count);
+    returnedVec.emplace_back(step);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:12]: redundant

File: homework/generate-sequence/generateSequence.hpp [hunk 2]
```diff
@@ -3,5 +3,18 @@
...
+    
+    for(int i=1; i< count; ++i)
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:14]: Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else etc...

File: homework/generate-sequence/generateSequence.hpp [hunk 3]
```diff
@@ -3,5 +3,18 @@
...
+    {
+        returnedVec.emplace_back(returnedVec[i-1]+step);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:16]: spaces!

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**